client/src/components/Visualization/charts/ScatterPlot.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Description: exports a react/victory object of 
+ *              a scatterplot with the top X most prolific advertisers
+ *              across 2018, 2019, 2020, as computed in `aggregates`
+ *              firestore collection. 
+ * Date: 7/21/2020
+ * Author: Robbie Marcus
+ */
+
+import { database } from '../../../firebase/firebase';
+import React, { useState, useEffect } from 'react';
+import { VictoryTheme, 
+         VictoryScatter, 
+         VictoryChart, 
+         VictoryAxis, 
+         VictoryTooltip, 
+         VictoryLabel } from 'victory';
+
+/**
+ * Creates a hexidecimal color string from a string input. 
+ * Taken from here: `https://stackoverflow.com/a/16348977`.
+ * Loosely cleaned up for style and comments.
+ * This *should* return a valid hash, but if it doesn't,
+ * the browser will just display `#000`.
+ * @param {string} str input string 
+ * @return {string}
+ */
+function stringToColor(str) {
```

Comment:
Consider putting this function in a utility class since it isn't directly related to the Scatter Plot and can be generalized for other visualization components